### DISCUSSION TITLE
Import of FHIR ValueSet

### DISCUSSION THREAD

**Discussion by xlemassacre** [b2ihealthcare/snow-owl#939] [Q&A] [answered]
Hi,

i'm trying to get a FHIR Profile with custom FHIR CodeSystems and ValueSets validated by Snow Owl.

It's possible to import CodeSystem, but ValueSet is not supported as FHIR POST. Is there another way to import those FHIR resources?

**Comment by cmark** [comment, marked as answer]
Hi @tpawork,

While the CodeSystem resource is supported in the open-source version, Value Sets and Concept Maps are only available in the pro version.
If you are interested please contact us [here](mailto:info@b2i.sg).

Cheers,
Mark

**Comment by cmark** [comment]
_Converting to discussions..._
